modules/serverless-python-function-return-values.adoc

```diff
@@ -6,7 +6,7 @@
 [id="serverless-python-function-return-values_{context}"]
 = Python function return values
 
-Functions can return any value supported by link:https://flask.palletsprojects.com/en/1.1.x/quickstart/#about-responses[Flask] because the invocation framework proxies these values directly to the Flask server.
+Functions can return any value supported by link:https://flask.palletsprojects.com/en/1.1.x/quickstart/#about-responses[Flask]. That is because the invocation framework proxies these values directly to the Flask server.
```

Comment:
```suggestion
Functions can return any value supported by link:https://flask.palletsprojects.com/en/1.1.x/quickstart/#about-responses[Flask]. This is because the invocation framework proxies these values directly to the Flask server.
```